css/signin.css

```diff
@@ -160,6 +258,13 @@ body {
     display: none; /* Controlled via JavaScript */
     text-align: center;
     font-weight: 500;
+    position: absolute;
+    top: 20px;
+    left: 50%;
```

Comment:
`left: 50%;` , `max-width:450px;` , `width: calc(100% -40px);` should all be using rem units!